src/main/java/database/managers/PlayerCreatorManager.java

```diff
@@ -0,0 +1,110 @@
+package database.managers;
+
+import interfaces.State;
+import playercreation.PlayerCreatorInteractor;
+import playercreation.PlayerQuestion;
+import playercreation.PlayerQuestionState;
+import playercreation.PlayerConfirmState;
+import core.StateManager;
+
+import java.util.ArrayList;
+
+public class PlayerCreatorManager extends StateManager {
+    /** A controller class that helps create a new Player character at the beginning of the game.
+     * Attributes:
+     * currPlayerQuestion: The current question about the Player character being asked of the user. An enum of
+     *                     type PlayerQuestion.
+     * completedQuestions: A List of completed PlayerQuestions that have already been filled by the user. Does not
+     *                     include CONFIRM or COMPLETE.
+     * creatorInteractor: A PlayerCreatorInteractor that stores information about the Player until all questions have
+     *                    been answered by the user.
+     */
+
+    private PlayerQuestion currPlayerQuestion;
+    private final ArrayList<PlayerQuestion> completedQuestions;
```

Comment:
it looks like you arent using other properties of the arrayList other than size, is there a problem with replacing this with an integer or even just associating each currPlayerQuestion with a number?